pkg/cli/install_test.go

```diff
@@ -0,0 +1,165 @@
+package cli
+
+import (
+	"io"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/acorn-io/acorn/pkg/cli/testdata"
+	"github.com/acorn-io/acorn/pkg/client"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInstall(t *testing.T) {
+	type fields struct {
+		Quiet  bool
+		Output string
+		All    bool
+	}
+	type args struct {
+		cmd    *cobra.Command
+		args   []string
+		client *testdata.MockClient
+	}
+	var _, w, _ = os.Pipe()
+	tests := []struct {
+		name           string
+		fields         fields
+		args           args
+		wantErr        bool
+		wantOut        []string
+		commandContext client.CommandContext
```

Comment:
Since this field is the same for each test case, does it make sense to remove the `commandContext` field here and have a single `commandContext` for the whole test?